pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -406,6 +409,7 @@ var _ = Describe("Pod Network", func() {
 					mockNetwork.EXPECT().HasNatIptables(proto).Return(true).Times(2)
 				}
 				mockNetwork.EXPECT().IsIpv6Enabled(podInterface).Return(true, nil).Times(3)
```

Comment:
can you explain  why is this function expected to be called 3 `Times`?

Reply:
i didnt touch this function in this PR